docs/zh_cn/hadoop_java_sdk.md

```diff
@@ -96,68 +107,74 @@ JuiceFS Hadoop Java SDK 同时兼容 Hadoop 2.x 以及 Hadoop 3.x 环境，以
 | Presto | `${PRESTO_HOME}/plugin/hive-hadoop2` |
 | Flink  | `${FLINK_HOME}/lib`                  |
 
-## 配置参数
+### 客户端配置参数
+
+请参考以下表格设置 JuiceFS 文件系统相关参数，并写入配置文件，一般是 `core-site.xml`。
 
-### 核心配置
+#### 核心配置
 
-| 配置项                           | 默认值                       | 描述                                                                                                                                                                                                         |
-| ------------------------------   | --------------------------   | ------------------------------------------------------------                                                                                                                                                 |
-| `fs.jfs.impl`                    | `io.juicefs.JuiceFileSystem` | 指定 `jfs://` 这个存储类型所使用的实现。JuiceFS 支持修改 scheme，例如想要使用 `cfs://` 作为 scheme，则将 `fs.cfs.impl` 的实现修改为此配置即可，当使用 `cfs://` 访问数据的时候，仍然是访问的 JuiceFS 的数据。 |
-| `fs.AbstractFileSystem.jfs.impl` | `io.juicefs.JuiceFS`         |                                                                                                                                                                                                              |
-| `juicefs.meta`                   |                              | Redis 地址，格式为 `redis://<user>:<password>@<host>:<port>/<db>`。                                                                                                                                          |
-| `juicefs.accesskey`              |                              | 对象存储的访问 ID（Access Key ID）。如果计算节点已经有访问对象存储的权限，则无需提供。请查看[这个文档](how_to_setup_object_storage.md)了解如何获取不同对象存储的 access key。                          |
```

Comment:
deleted?

Reply:
I have discussed with @tangyoupeng tang, he checks the code, the file system needs to be formatted before add to juicefs java client, and the access key of the object storage is no longer needed here.